LibraryApp/src/components/books-list.js

```diff
@@ -54,9 +94,13 @@ class BooksList extends React.Component {
   }
 
   selectBook(book) {
-    const {setSelectedBook} = this.props;
+    const {setSelectedBook, navigation} = this.props;
     setSelectedBook(book);
-    this.navigateToForm();
+    navigation.navigate(showBookDetails, {
+      params: {
```

Comment:
do we need params?

Reply:
Removed. Working on removing params as well.